README.md

```diff
@@ -0,0 +1,169 @@
+  - [Link to the back-end repo](#link-to-the-back-end-repo)
+  - [Link to the kanbarn board](#link-to-the-kanbarn-board)
+  - [Link to the kanbarn board at its initial state](#link-to-the-kanbarn-board-at-its-initial-state)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏Acknowledgments ](#acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 [Appointment_booker_front_end] <a name="about-project"></a>
+
+**[Appointment_booker_front_end]** is a frontend react app for our doctors appointment booking application. It shows the features of the app and provides a user with better experience and interface
+
+## [Link to the back-end repo](https://github.com/0sugo/appointment_booker_backend.git)
+
+## [Link to the kanbarn board](https://github.com/0sugo/appointment_booker_backend/projects/1)
+
+## [Link to the kanbarn board at its initial state](./public/kanban%20board.png)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>React</summary>
+  <ul>
+    <li><a>https://react.org/</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Redux</summary>
+  <ul>
+    <li><a>https://redux.js.org/</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Render available doctors]**
+- **[Add doctors]**
+- **[Delete doctors]**
+- **[Reserve an appointment with a specific doctor]**
+- **[View all appointments for a specific user]**
+- **[Delete an appointment with a specific doctor]**
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need a web browser.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone https://github.com/0sugo/appointment_booker_frontend.git
+  cd appointment_booker_frontend
+```
+
+### Install
+
+Install this project with:
+
+```
+  cd appointment_booker_frontend
+  npm install
+```
+```
+  npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x @babel/preset-env@7.x  @babel/preset-react@7.x
+```
+```
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+npm test
```

Comment:
- [ ] When I run the frontend tests according to instructions, I got one failing:

![image](https://github.com/0sugo/appointment_booker_frontend/assets/57421823/897e1565-07c3-4cdf-a014-96a55f9caa4c)

Kindly, make it so all your tests pass. 💪